.gitlab/source_test/ebpf.yml

```diff
@@ -16,6 +16,10 @@
   before_script:
     - !reference [.retrieve_linux_go_deps]
     - !reference [.retrieve_linux_go_tools_deps]
+    # XXX: system-probe images does not run `entrypoint.sh` which runs `source /root/.bashrc`.
+    # Since the path of Go binary is added into PATH in the bashrc, we need to run it at some point.
+    # This is a workaround until I understand why we don't wanna run `entrypoint.sh` for system-probe. 
+    - source /root/.bashrc
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Would it be better to put this at the top of the `before_script` section? I think it might be proper

Reply:
I don't have a strong opinion here, I'll first wait feedback from ebpf team as I feel this should not be done here. Instead we might want to use `entrypoint.sh` in system-probe buildimages (but I guess there is a reason why we don't already do it)

Reply:
I've removed this part in the end